keras_cv/layers/object_detection/anchor_generator_test.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+from absl.testing import parameterized
+
+from keras_cv import layers as cv_layers
+
+
+class AnchorGeneratorTest(tf.test.TestCase, parameterized.TestCase):
```

Comment:
I think we should do numerical verification, since there isn't any randomness involved here.

Reply:
Good point, I'll add a test case with carefully crafted values.